### PR TITLE
Backport 5.5.1

### PR DESCRIPTION
**Pull request by sfan5**
**DO NOT MERGE THIS PR, IT IS ONLY FOR CHANGE TRACKING**

Notable:
* 1e4d6672be35e075de6add3d4d4e97793a911efc not sure if too much of a behaviour change, but included anyway
* 1ee37148a8072fe6350124cd51c812c3d3fb069a skipped due to very low impact
* c31b3017222edd6e93bdeb02f05a3df7b6b23a1a contains a bugfix!
* ~~7db751df3bc4e3b3aff80cdacd2883f3a7a0940b could not be backported because it uses a newly introduced function~~

### CODE REVIEW

**Comment by pecksin** [comment]
#11969 could be a good idea, to help avoid the possibility of old_server + old_client + new_mods

**By itself** it does not require a protocol bump.

**Comment by sfan5** [comment]
> #11969 could be a good idea, to help avoid the possibility of old_server + old_client + new_mods

Just the client-side fix you mean?

**Comment by pecksin** [comment]
@sfan5 

The whole patch, server too.

Without a proto bump, this patch will simply treat everything as an old client, which is fine since there is no new functionality yet anyway, and will prevent _actual_ old clients from crashing/getting weird in case some server later updates its mods while still using 5.5.1

File: android/build.gradle [hunk 1]
```diff
@@ -4,7 +4,7 @@ project.ext.set("versionMajor", 5)      // Version Major
 project.ext.set("versionMinor", 5)      // Version Minor
 project.ext.set("versionPatch", 0)      // Version Patch
```
Comment (rubenwardy) [inline android/build.gradle:5]: Should be 1

**Pull request merged**